src/models/Jalasoft.TeamUp.Resumes.Models/Skill.cs

```diff
@@ -6,6 +6,6 @@ public class Skill
     {
         public Guid Id { get; set; }
```

Comment:
so far I understand, this should be string, right @ela-ocana-jalafund

Reply:
Yes, Ela told me, it is already corrected